src/seq_analysis/kmer.rs

```diff
@@ -113,6 +127,39 @@ pub fn next_dnakmer(kmer: DNAKMer, k: usize, b: u8) -> DNAKMer {
     return DNAKMer(result); 
 }
 
+pub fn dna_complement(base: u8) -> u8 {
```

Comment:
Hi,

Why you didn't use the complement fonction implement in the alphabet module ? https://github.com/rust-bio/rust-bio/blob/master/src/alphabets/dna.rs#L41